### PR TITLE
Update metadata related BBEs

### PR DESCRIPTION
**Pull request by lasinicl**
## Purpose
$title

### CODE REVIEW

File: examples/annotations/annotations.bal [hunk 1]
```diff
@@ -12,8 +12,8 @@ public function transform(string s) returns string {
 public function main() {
     // The `@strand` annotation applies to the `start` action.
     future<int> fut = @strand {thread: "any"} start foo();
```
Comment (KRVPerera) [inline examples/annotations/annotations.bal:14]: strand annotation is found in this BBE too.

File: examples/annotations/annotations.bal [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import ballerina/io;
 
-// The `@display` annotation applies to the transform function.
+// The `@display` annotation applies to the `transform` function.
```
Comment (praneesha) [inline examples/annotations/annotations.bal:3]:
```suggestion
// The `@display` annotation is applied to the `transform` function.
```

File: examples/annotations/annotations.md [hunk 3]
```diff
@@ -1,9 +1,6 @@
 # Annotations
 
-Annotations start with `@tag` and they come before what they apply to.
-Unprefixed tags refer to standard platform-defined annotations and
-prefixed tags refer to annotations declared in modules.
-`@tag` can be followed by record constructor expression.
+Annotations start with `@tag` and they come before what they apply to. Unprefixed tags refer to standard platform-defined annotations and prefixed tags refer to annotations declared in modules. `@tag` can be followed by a record constructor expression.
```
Comment (praneesha) [inline examples/annotations/annotations.md:3]:
```suggestion
Annotations start with the `@tag` and they come before what they apply to. Unprefixed tags refer to standard platform-defined annotations and prefixed tags refer to annotations declared in modules. `@tag` can be followed by a record constructor expression.
```

**Review by KRVPerera — APPROVED** [review]
LGTM

**Review by praneesha — APPROVED** [review]

**Pull request merged**